GetJobProfiles/Models/Recipe/Parts/GraphSyncPart.cs

```diff
@@ -4,7 +4,7 @@ namespace GetJobProfiles.Models.Recipe.Parts
 {
     public class GraphSyncPart
     {
-        public GraphSyncPart(string contentType) => Text = $"http://nationalcareers.service.gov.uk/{contentType.ToLowerInvariant()}/{Guid.NewGuid()}";
+        public GraphSyncPart(string contentType) => Text = $"[js:configuration('ContentApiPrefix') +" + "'/" + contentType.ToLowerInvariant() + "/" + Guid.NewGuid() + "'" + "]";
```

Comment:
Not a biggie, but could still use interpolation here, rather than concatenation.